ofdb-boundary/src/lib.rs

```diff
@@ -1213,3 +1213,24 @@ impl From<Entry> for UpdatePlace {
         }
     }
 }
+
+#[derive(Serialize, Deserialize)]
+#[cfg_attr(feature = "extra-derive", derive(Debug, Clone, PartialEq))]
+pub struct Error {
```

Comment:
I think this `Error` struct should not be part of this PR.

Reply:
yeah, a rebasing mistake. done.